webpack.config.js

```diff
@@ -50,7 +61,7 @@ module.exports = () => {
               {
                 loader: 'css-loader',
                 options: {
-                  modules: true,
+                  modules: false,
```

Comment:
Could you please move it into separate PR?

Reply:
reverted from this pr